### ISSUE TITLE
Regression found for suggestions without tld

### ISSUE THREAD

**Issue by caphun** [mailcheck/mailcheck#139]
@derrickko 

One of the recent merges, either #133 or #137 [**update:** sorry, nothing to do with 137 as that's just a text change] has a regression. If you type `a@gm` it will suggest `a@com` and `a@gmail` comes back with no suggestions where as it use to return `a@gmail.com`

This is happening on the master branch. If you apply my fix in #138 it will resolve the first issue but the second issue still remains.

**Comment by ferreiro** [comment]
yes! the problem is with #133 I mentioned in this PR: https://github.com/ZooTools/email-spell-checker/pull/3

----

FYI, I rewrote it entirely and update it for 2022 and onward: https://github.com/ZooTools/email-spell-checker

- 💙 Written in TypeScript and removed jQuery
- ✅ Fixed a couple of bugs like [this](https://github.com/ZooTools/email-spell-checker/pull/3) or [this](https://github.com/ZooTools/email-spell-checker/pull/4)
- 🚀 Reduced bundle size to <2KB. 
- ✨ Update TLDs (69+) and added modern startup domains (like .io, .so, .xyz or .dev)
- 🙏 Implemented suggestions that people made in this repo that were ignored.

Link: https://github.com/ZooTools/email-spell-checker

Come check it out and give it a ⭐️ for the effort.

**Comment by skyfallsin** [comment]
Closing as resolved. Exact, unambiguous missing endings such as `sample@gmail` now suggest `sample@gmail.com`; partial names such as `a@gm` are not guessed.
